### ISSUE TITLE
typedef enum

### ISSUE THREAD

**Issue by Konstantin8105** [elliotchance/c2go#246]
Example of C code:
```c
typedef enum {RANDOM, IMMEDIATE, SEARCH} strategy;
strategy my_strategy = IMMEDIATE;
```

**Comment by Konstantin8105** [comment]
Example:
```c
 typedef enum {
    a, b, c
 } MY_TYPE;
 typedef struct {
    MY_TYPE myEnumVar;
 } MY_STRUCT;
 MY_STRUCT myStruct = {a};
```

**Comment by elliotchance** [comment]
Was this already fixed in #252?

**Comment by Konstantin8105** [comment]
No, In according to https://github.com/elliotchance/c2go/blob/master/tests/enum.c . we haven't implementation `enum` with `typedef`.
